internal/status/route_status.go

```diff
@@ -286,10 +290,6 @@ func updateRouteObject(mRoute *routev1.Route, updateOption UpdateOptions, retryN
 	if len(updateOption.Vip) == 0 {
 		return nil
 	}
-	if updateOption.ServiceMetadata.InsecureEdgeTermAllow {
```

Comment:
@swathinsankaran: Will removing this might create issue in some other code flow? Like WCP?

Reply:
AKO running in WCP also faces the same issue of no status when the insecure edge termination policy is set as Allow. This would fix it.